build.gradle

```diff
@@ -1,15 +1,18 @@
 buildscript {
 
+    ext {
+        kotlin_version = '1.4.20'
+    }
     repositories {
         gradlePluginPortal()
         jcenter()
         google()
         mavenCentral()
     }
     dependencies {
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.10"
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:1.4.20"
```

Comment:
Will be better to have one single define for kotlin version?

Reply:
I think we have that in the shared gradle file, no?